infrastructure/variables.tf

```diff
@@ -77,4 +77,9 @@ variable asp_rg_override {
 variable vault_name_override {
   description = "Vault Name"
   default = ""
+}
+
+variable "strategic_policies_privateIpsFilterPattern" {
```

Comment:
The TF variable name doesn't need to match the application property name. Maybe we can simplify to:
```
vnet_private_ip_pattern
```

Reply:
Conversely, I think retaining the same name make it much easier to identify mistakes and issues in the marshalling of variables through the provisioning tasks.